src/components/Column/Column.scss

```diff
@@ -58,10 +63,15 @@
   line-height: $line-height--large;
 }
 
+.column__notes-wrapper {
+  overflow: auto;
+}
+
 .column__note-list {
-  padding: 0;
+  margin: 0;
+  padding: 0 $padding--large;
   display: grid;
-    grid-template-columns: repeat(auto-fill,minmax(260px,1fr));
+    grid-template-columns: repeat(auto-fill);
```

Comment:
Regarding to my Chrome DevTools that's a `Invalid property value`

Reply:
I reversed the changes, they have no effect

Reply:
Problem seems to be back